src/lib.rs

```diff
@@ -121,8 +127,12 @@ macro_rules! error {
 /// * The visibility of all the items in the original `impl` must be identical.
 #[proc_macro_attribute]
 pub fn ext(args: TokenStream, input: TokenStream) -> TokenStream {
+    let ext_ident = match syn::parse_macro_input!(args) {
+        None => format_ident!("__ExtTrait{}", hash(&input)),
```

Comment:
See https://github.com/taiki-e/auto_enums/pull/60 for reasons to use the hash value of the input AST.